test/L2/paused/L2VotingPowerPaused.t.sol

```diff
@@ -0,0 +1,110 @@
+import { IVotes } from "@openzeppelin/contracts/governance/utils/IVotes.sol";
+import { ERC1967Proxy } from "@openzeppelin/contracts/proxy/ERC1967/ERC1967Proxy.sol";
+import { OwnableUpgradeable } from "@openzeppelin-upgradeable/contracts/access/OwnableUpgradeable.sol";
+import { IAccessControl } from "@openzeppelin/contracts/access/IAccessControl.sol";
+import { IERC20Errors } from "@openzeppelin/contracts/interfaces/draft-IERC6093.sol";
+import { IERC20 } from "@openzeppelin/contracts/token/ERC20/IERC20.sol";
+import { Test, console } from "forge-std/Test.sol";
+import { IL2LockingPosition } from "src/interfaces/L2/IL2LockingPosition.sol";
+import { L2VotingPower } from "src/L2/L2VotingPower.sol";
+import { L2VotingPowerPaused } from "src/L2/paused/L2VotingPowerPaused.sol";
+import { Utils } from "script/contracts/Utils.sol";
+
+contract MockL2VotingPowerV2 is L2VotingPower {
+    uint256 public testNumber;
+
+    function initializeV2(uint256 _testNumber) public reinitializer(3) {
+        version = "2.0.0";
+        testNumber = _testNumber;
+    }
+
+    function onlyV2() public pure returns (string memory) {
+        return "Only L2VotingPowerV2 have this function";
+    }
+}
+
+contract L2VotingPowerPausedTest is Test {
+    Utils public utils;
+    L2VotingPower public l2VotingPowerImplementation;
+    L2VotingPower public l2VotingPower;
+
+    address lockingPositionContractAddress;
+
+    function setUp() public {
+        utils = new Utils();
+
+        // set initial values
+        lockingPositionContractAddress = address(0xdeadbeefdeadbeefdeadbeef);
+
+        console.log("L2VotingPowerTest address is: %s", address(this));
+
+        // deploy L2VotingPower Implementation contract
+        l2VotingPowerImplementation = new L2VotingPower();
+
+        // deploy L2VotingPower contract via proxy and initialize it at the same time
+        l2VotingPower = L2VotingPower(
+            address(
+                new ERC1967Proxy(
+                    address(l2VotingPowerImplementation),
+                    abi.encodeWithSelector(l2VotingPower.initialize.selector, lockingPositionContractAddress)
+                )
+            )
+        );
+
+        assertEq(l2VotingPower.lockingPositionAddress(), lockingPositionContractAddress);
+        assertEq(l2VotingPower.version(), "1.0.0");
+        assertEq(l2VotingPower.name(), "Lisk Voting Power");
+        assertEq(l2VotingPower.symbol(), "vpLSK");
+
+        // Upgrade from L2VotingPower to L2VotingPowerPaused
+        L2VotingPowerPaused l2VotingPowerPausedImplementation = new L2VotingPowerPaused();
+        l2VotingPower.upgradeToAndCall(address(l2VotingPowerPausedImplementation), "");
+
+        // Call initializePaused with new interface
+        L2VotingPowerPaused(address(l2VotingPower)).initializePaused();
+
+        // l2VotingPower pointing to paused contract
+        assertEq(l2VotingPower.version(), "1.0.0-paused");
+    }
+
+    function test_adjustVotingPower_Paused() public {
```

Comment:
As stated above, this function should not be paused. And it may even be worth to test that this function indeed still works as it's crucial for staking. But if you think it's not needed, then just remove the test.

Reply:
Changed to test `adjustVotingPower` is not paused after changing to paused contract